handycon.py

```diff
@@ -120,7 +122,7 @@ def id_system():
 
     # Identify the current device type. Kill script if not compatible.
     system_id = open("/sys/devices/virtual/dmi/id/product_name", "r").read().strip()
-    system_cpu = subprocess.check_output("lscpu | grep \"Vendor ID\" | cut -d : -f 2 | xargs", shell=True, universal_newlines=True).strip()
+    system_cpu = platform.processor()
```

Comment:
We aren't using this anymore actually, we can get rid of it and the platform import.